components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -1905,6 +1906,10 @@ def full_viewer(request, iid, conn=None, **kwargs):
     """
 
     rid = getImgDetailsFromReq(request)
+
+    interpolate = toBoolean(conn.getConfigService().getConfigValue(
```

Comment:
This will call server each time you open a viewer. That is very inefficient as it is not image specific. Could you load it once and add it to server exactly as https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/decorators.py#L254